src/api/controllers/habit.controllers.js

```
@@ -1 +1,31 @@
-// 요청 파싱(params/query/body) + 입력 검증 결과 처리
+// 요청 파싱(params/query) + 입력 검증 + 서비스 호출 + 에러 변환
+
+import getTodayHabitsService from '../services/habit.services.js';
```

ESLint

[error] Unexpected use of file extension "js" for "../services/habit.services.js"

(import/extensions)